Automattic/jetpack
# Blog Stats Block: Remove Check for WP.com Sites

Fixes #35437

## Proposed changes:

* Removes the check that stops the block from registering on WordPress.com sites.

## Jetpack product discussion


See here: https://github.com/Automattic/jetpack/pull/34557#issuecomment-1952538401

> We should indeed be able to remove that check now.

@jeherve just to confirm, does this only apply to the Blog Stats block? Can we remove the same check for the Top Posts & Pages block now too or does that endpoint still need to be added? 

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:
Not quite sure how to test it, so would appreciate some guidance on this. I suppose it's a matter of trying this on a WordPress.com environment and verifying that the stats load when inserting the Blog Stats block.

Note: the block is in beta, so you'll need to enable Beta blocks first.

cc @kangzj
